spec/spec_helper.rb

```diff
@@ -24,7 +24,7 @@ module Mock
   class Reporter < Datadog::Notifications::Reporter
     def timing(stat, ms, opts={}); super(stat, 333, opts); end
     def flush_buffer; end
-    alias :send_stat :send_to_buffer
+    @should_batch = true
```

Comment:
Existing `alias :send_stat :send_to_buffer` caused rspec to fail.
I have copied the change from here:
https://github.com/DataDog/dogstatsd-ruby/commit/c41aa6cf16fd281b6c4daabd1d993f0d8dda412f#diff-f7b44886d16ab8eb23deaed048ad0b10L302